modules/peridynamics/src/auxkernels/MechanicalStretchPD.C

```diff
@@ -16,7 +16,7 @@ InputParameters
 validParams<MechanicalStretchPD>()
 {
   InputParameters params = validParams<AuxKernelBasePD>();
-  params.addClassDescription("Class for outputing bond mechanical stretch value");
+  params.addClassDescription("Class for outputing bond mechanical stretch");
   params.set<ExecFlagEnum>("execute_on") = EXEC_TIMESTEP_END;
```

Comment:
Can't you just use MaterialRealAux for this purpose?  Also, I'm a little hesitant about setting a nonstandard value for 'execute_on' -- if the user wants it just for output purposes, I think I would expect them to set this.